report/43_Lessons-Maintaince.md

```diff
@@ -1 +1,9 @@
-maintenance
+## Maintenance
+To facilitate maintenance of our the MiniTwit system without we have multiple CI tools which checked that the system was still in working order after changes. These were the Unit Test, API simulator, SonarCloud and Google's Lighthouse. 
+The last of which, Lighthouse, inadvertently showed us we had a issue when merging code to master. 
```

Comment:
"showed us we had an issue" men næste linje skriver du "not getting spotted". Husk forresten at undgå vi